modules/virt-customizing-storage-profile.adoc

```diff
@@ -19,21 +22,21 @@ If you create a data volume and omit YAML attributes and these attributes are no
 ====
 
 .Procedure
-. Edit the storage profile. In this example, the provisioner is not recognized by the CDI:
+. Edit the storage profile. In this example, the provisioner is not recognized by CDI:
 +
-[source,terminal]
+[source,YAML]
 ----
 $ oc edit -n openshift-cnv storageprofile <storage_class>
 ----
 +
-.Example output
-[source,terminal]
+.Example storage profile
+[source,YAML]
 ----
 apiVersion: cdi.kubevirt.io/v1beta1
 kind: StorageProfile
 metadata:
   name: <some_unknown_provisioner_class>
-    ...
+#   ...
 spec: {}
 status:
   provisioner: <some_unknown_provisioner>
```

Comment:
Github won't let me comment on the exact lines, but for this code block in step 2, please change the title to `.Example storage profile` and use `[source,yaml]`

Reply:
Done.